core/arch/interp.c

```diff
@@ -3744,6 +3744,21 @@ build_bb_ilist(dcontext_t *dcontext, build_bb_t *bb)
             break;
         }
 
+#ifdef X86
+        if (my_dcontext != NULL && instr_is_syscall(bb->instr) &&
+            instr_get_opcode(bb->instr) == OP_int &&
```

Comment:
It looks like I should compute both do_syscall (int and other) method at initialization since the do_syscall method is written in the .data section, so it cannot be updated on Windows